lib/spack/docs/environments.rst

```diff
@@ -1000,6 +1000,28 @@ For example, the following environment has three root packages:
 This allows for a much-needed reduction in redundancy between packages
 and constraints.
 
+-------------------------------
+Modifying Environment Variables
+-------------------------------
+
+Spack Environments can modify environment variables when activated.  The environment can be
```

Comment:
```suggestion
Spack Environments can modify the active shell's environment variables when activated.  The environment can be
```
I feel like just stating environment variables is a little too ambiguous here.

Reply:
Thanks thats a good clarification